README.md

```diff
@@ -234,15 +295,17 @@ concurrency:
 
 jobs:
   multi-approvers:
-    uses: 'abcxyz/pkg/.github/workflows/multi-approvers.yml@main'
-    with:
-      org-members-path: 'abcxyz/pkg/main/.github/workflows/members.json'
+    if: |-
```

Comment:
Is this needed if the workflow target is `on: pull_request, pull_request_review`?

Reply:
You're right. This is not necessary. Removed in two places.